### PR TITLE
[WiP] Advertise community.brave.com

### PR DESCRIPTION
**Pull request by luixxiul**
- [ ] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [ ] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [ ] Ran `git rebase -i` to squash commits (if needed).

Auditors: @bsclifton, @alexwykoff

Test Plan:
1. Make sure the anchor link to the community (discourse) on README works
2. Make sure "Submit Feedback..." in the Menu opens the community in a new tab
3. Make sure "Send us feedback..." on about:preferences opens it in a new tab

### CODE REVIEW

**Comment by luixxiul** [comment]
Some details should be added on README.md.

**Comment by alexwykoff** [comment]
Please don't forget the hamburger (now kabob) menu links.
'Help' -> 'Report an Issue' should link to community as well for now.

A spec is in progress for the new help actions, but this is a good stopgap. Thanks for getting it started!

**Comment by luixxiul** [comment]
@alexwykoff then you would have the two items in the same submenu which are linked to the same page, which is not expected? In this case either of it should be removed.

**Comment by alexwykoff** [comment]
You could deep link to the appropriate threads in community, but yes, I did suggest double-linking because it is a stopgap, not a full solution.

File: app/common/commonMenu.js [hunk 1]
```diff
@@ -10,7 +10,8 @@ const Immutable = require('immutable')
 const locale = require('../../js/l10n')
 const settings = require('../../js/constants/settings')
 const getSetting = require('../../js/settings').getSetting
-const issuesUrl = 'https://github.com/brave/browser-laptop/issues'
+// const issuesUrl = 'https://github.com/brave/browser-laptop/issues'
```
Comment (bsclifton) [inline app/common/commonMenu.js]: I'd suggest just deleting this; comments usually just end up cluttering and we can always look at the previous revision :smile:

**Comment by bsclifton** [comment]
++

**Comment by luixxiul** [comment]
@bsclifton would you please add information about the community on README.md which sounds natural to native speakers?

**Comment by luixxiul** [comment]
Rebased.

**Comment by luixxiul** [comment]
Note: `reportAnIssueMenuItem` and `submitFeedbackMenuItem` in commonMenu.js and menu.js should be merged and replaced later

**Comment by alexwykoff** [comment]
lgtm!

**Comment by bbondy** [comment]
Added an issue for it here:
https://github.com/brave/browser-laptop/issues/6179

**Comment by luixxiul** [comment]
@bbondy thanks for doing that!

**Pull request merged**